qaframework-bdd-tests/src/test/java/org/openmrs/contrib/qaframework/automation/FindPatientSteps.java

```diff
@@ -64,7 +64,7 @@ public void returnResults() {
 		assertNotNull(firstPatientIdentifier);
 	}
 
-	@And("User clicks on first patient")
+	@And("User clicks on returned patient")
 	public void clickFirstPatient() {
```

Comment:
isn't the method name changing also?

Reply:
The method name has not changed but we have changed the gherkin scenario(user story). please cross-check again especially all those i changed.

Reply:
Does the method name for the test still remain `clickFirstPatient()`?

Reply:
The patient is being searched by identifier hence returning only one patient, don't you think there is a need to rename the method?

Reply:
Resolved thanks.